### PR TITLE
Support for parameter-less and metadata-less models 

### PR DESCRIPTION
**Pull request by amontagne**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,5 +19,8 @@ venv.bak/
 *.idea
 *.iml
 
-# Outputs
-outputs/
\ No newline at end of file
+# Data
+data/
```
Comment (m-prl) [inline .gitignore:23]: change data/ to /data/, as we don't want to ignore data directory in folders such as tests (https://www.atlassian.com/fr/git/tutorials/saving-changes/gitignore)

File: apparun/impact_methods.py [hunk 2]
```diff
@@ -9,8 +9,6 @@ class MethodFullName(str, Enum):
     So far, only some ReCiPe methods, and all EF v3.0 impact methods are supported.
     """
 
-    _settings_ = NoAlias
```
Comment (m-prl) [inline apparun/impact_methods.py:12]: why removing?

File: tests/unit_tests.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+from apparun.results import get_result
+
+
+def test_impact_tree_calculation():
+    impact_model = ImpactModel().from_yaml("data/noparam_system.yaml")
+    lcia_scores = impact_model.get_scores()
+    scores = sorted(lcia_scores.scores.items())
+    assert dict(scores) == {'EFV3_CLIMATE_CHANGE': 1.3830513221576728e-06, 'EFV3_ECOTOXICITY_FRESHWATER': 4.365302017373894e-12, 'EFV3_LAND_USE': 0.0}
+
+def test_impact_calculation_result():
```
Comment (m-prl) [inline tests/unit_tests.py:18]: shouldn't the test contain an assertion?

**Review by m-prl — CHANGES_REQUESTED** [review]

File: apparun/impact_methods.py (hunk 2 above)
Comment (amontagne) [reply apparun/impact_methods.py:12]: to enable by-value lookups
`NoAlias` is not required as there are no value duplicates in `MethodFullName`

File: tests/unit_tests.py (hunk 3 above)
Comment (amontagne) [reply tests/unit_tests.py:18]: I could test that the files are created

**Pull request merged**